In [21]:
import pandas as pd
# Abro el archivo CSV con el encoding latin-1
df = pd.read_csv('GSAF5.csv', encoding='latin-1')
#Muestro las 10 primeras filas de mi Data Frame
df.head(5)


Case Number       Date  Year        Type    Country      Area  \
0  2016.09.18.c  18-Sep-16  2016  Unprovoked        USA   Florida   
1  2016.09.18.b  18-Sep-16  2016  Unprovoked        USA   Florida   
2  2016.09.18.a  18-Sep-16  2016  Unprovoked        USA   Florida   
3    2016.09.17  17-Sep-16  2016  Unprovoked  AUSTRALIA  Victoria   
4    2016.09.15  16-Sep-16  2016  Unprovoked  AUSTRALIA  Victoria   

                           Location Activity             Name Sex   \
0  New Smyrna Beach, Volusia County  Surfing             male    M   
1  New Smyrna Beach, Volusia County  Surfing   Chucky Luciano    M   
2  New Smyrna Beach, Volusia County  Surfing             male    M   
3                  Thirteenth Beach  Surfing  Rory Angiolella    M   
4                       Bells Beach  Surfing             male    M   

      ...       Species        Investigator or Source  \
0     ...            NaN  Orlando Sentinel, 9/19/2016   
1     ...            NaN  Orlando Sentinel, 9/19/2016   
2     ...            NaN  Orlando Sentinel, 9/19/2016   
3     ...            NaN           The Age, 9/18/2016   
4     ...      2 m shark           The Age, 9/16/2016   

                         pdf  \
0       2016.09.18.c-NSB.pdf   
1   2016.09.18.b-Luciano.pdf   
2       2016.09.18.a-NSB.pdf   
3  2016.09.17-Angiolella.pdf   
4  2016.09.16-BellsBeach.pdf   

                                        href formula  \
0  http://sharkattackfile.net/spreadsheets/pdf_di...   
1  http://sharkattackfile.net/spreadsheets/pdf_di...   
2  http://sharkattackfile.net/spreadsheets/pdf_di...   
3  http://sharkattackfile.net/spreadsheets/pdf_di...   
4  http://sharkattackfile.net/spreadsheets/pdf_di...   

                                                href Case Number.1  \
0  http://sharkattackfile.net/spreadsheets/pdf_di...  2016.09.18.c   
1  http://sharkattackfile.net/spreadsheets/pdf_di...  2016.09.18.b   
2  http://sharkattackfile.net/spreadsheets/pdf_di...  2016.09.18.a   
3  http://sharkattackfile.net/spreadsheets/pdf_di...    2016.09.17   
4  http://sharkattackfile.net/spreadsheets/pdf_di...    2016.09.16   

  Case Number.2 original order Unnamed: 22 Unnamed: 23  
0  2016.09.18.c           5993         NaN         NaN  
1  2016.09.18.b           5992         NaN         NaN  
2  2016.09.18.a           5991         NaN         NaN  
3    2016.09.17           5990         NaN         NaN  
4    2016.09.15           5989         NaN         NaN  

[5 rows x 24 columns]

Miramos el número total de columnas y filas que tiene mi data frame

In [23]:
df.shape

(5992, 24)

Comprobación del número de NULLS que hay en el data frame.

In [24]:
df.isnull().sum()


Case Number                  0
Date                         0
Year                         0
Type                         0
Country                     43
Area                       402
Location                   496
Activity                   527
Name                       200
Sex                        567
Age                       2681
Injury                      27
Fatal (Y/N)                 19
Time                      3213
Species                   2934
Investigator or Source      15
pdf                          0
href formula                 1
href                         3
Case Number.1                0
Case Number.2                0
original order               0
Unnamed: 22               5991
Unnamed: 23               5990
dtype: int64

Borranmos las columnas Unnamed 22 y Unnamed 23 (por tener prácticamente todos los valores en null) para ello empezaré haciendo una copia del data frame.

In [25]:
df_aux = df.copy()

Una vez tenemos la copia, borramos la columna.

In [26]:
df_aux.drop(columns=['Unnamed: 22','Unnamed: 23'],inplace=True)

Vemos como ya no tenemos las columnas que hemos borrado.

In [28]:
df_aux.head(5)

Case Number       Date  Year        Type    Country      Area  \
0  2016.09.18.c  18-Sep-16  2016  Unprovoked        USA   Florida   
1  2016.09.18.b  18-Sep-16  2016  Unprovoked        USA   Florida   
2  2016.09.18.a  18-Sep-16  2016  Unprovoked        USA   Florida   
3    2016.09.17  17-Sep-16  2016  Unprovoked  AUSTRALIA  Victoria   
4    2016.09.15  16-Sep-16  2016  Unprovoked  AUSTRALIA  Victoria   

                           Location Activity             Name Sex   \
0  New Smyrna Beach, Volusia County  Surfing             male    M   
1  New Smyrna Beach, Volusia County  Surfing   Chucky Luciano    M   
2  New Smyrna Beach, Volusia County  Surfing             male    M   
3                  Thirteenth Beach  Surfing  Rory Angiolella    M   
4                       Bells Beach  Surfing             male    M   

       ...       Fatal (Y/N)   Time   Species        Investigator or Source  \
0      ...                 N  13h00        NaN  Orlando Sentinel, 9/19/2016   
1      ...                 N  11h00        NaN  Orlando Sentinel, 9/19/2016   
2      ...                 N  10h43        NaN  Orlando Sentinel, 9/19/2016   
3      ...                 N    NaN        NaN           The Age, 9/18/2016   
4      ...                 N    NaN  2 m shark           The Age, 9/16/2016   

                         pdf  \
0       2016.09.18.c-NSB.pdf   
1   2016.09.18.b-Luciano.pdf   
2       2016.09.18.a-NSB.pdf   
3  2016.09.17-Angiolella.pdf   
4  2016.09.16-BellsBeach.pdf   

                                        href formula  \
0  http://sharkattackfile.net/spreadsheets/pdf_di...   
1  http://sharkattackfile.net/spreadsheets/pdf_di...   
2  http://sharkattackfile.net/spreadsheets/pdf_di...   
3  http://sharkattackfile.net/spreadsheets/pdf_di...   
4  http://sharkattackfile.net/spreadsheets/pdf_di...   

                                                href Case Number.1  \
0  http://sharkattackfile.net/spreadsheets/pdf_di...  2016.09.18.c   
1  http://sharkattackfile.net/spreadsheets/pdf_di...  2016.09.18.b   
2  http://sharkattackfile.net/spreadsheets/pdf_di...  2016.09.18.a   
3  http://sharkattackfile.net/spreadsheets/pdf_di...    2016.09.17   
4  http://sharkattackfile.net/spreadsheets/pdf_di...    2016.09.16   

  Case Number.2 original order  
0  2016.09.18.c           5993  
1  2016.09.18.b           5992  
2  2016.09.18.a           5991  
3    2016.09.17           5990  
4    2016.09.15           5989  

[5 rows x 22 columns]

Creamos una columna para ver si las columnas href formula y href son iguales y sumamos el numero de filas que son iguales en las dos columnas (5989, por lo que tenemos 54 filas diferentes).

In [29]:
df_aux.head()
df_aux['pdf_equals'] = (df_aux['href formula'] == df_aux['href'])
df_aux['pdf_equals'].sum()

5938

Creamos una columna para ver si las columnas Case Number.1 y Case Number.2 son iguales y sumamos el numero de filas que son iguales en las dos columnas (5981, por lo que tengo 11 filas diferentes).

In [30]:
df_aux.head()
df_aux['CaseNumber_equals'] = (df_aux['Case Number.1'] == df_aux['Case Number.2'])
df_aux['CaseNumber_equals'].sum()

5981

Mostramos las filas que son diferentes en href formula y href para ver si las diferencias son relevantes.

In [31]:
#df_aux[['Case Number.1','Case Number.2','CaseNumber_equals']]

In [32]:
df_aux[df_aux['pdf_equals']== False]

Case Number                  Date  Year          Type  \
20      2016.07.29             29-Jul-16  2016    Unprovoked   
27    2016.07.23.a             23-Jul-16  2016    Unprovoked   
61    2016.05.21.a             21-May-16  2016    Unprovoked   
107   2015.12.21.a             21-Dec-15  2015    Unprovoked   
114   2015.11.15.a             15-Nov-15  2015    Unprovoked   
134   2015.10.05.a             05-Oct-15  2015    Unprovoked   
180   2015.06.27.a             27-Jun-15  2015    Unprovoked   
193     2015.06.13             13-Jun-15  2015    Unprovoked   
232     2015.01.24             24-Jan-15  2015    Unprovoked   
262     2014.10.19             19-Oct-14  2014          Boat   
263     2014.10.18             18-Oct-14  2014    Unprovoked   
264     2014.10.17             17-Oct-14  2014    Unprovoked   
271   2014.10.03.b             03-Oct-14  2014          Boat   
272   2014.10.03.a             03-Oct-14  2014          Boat   
293     2014.08.12             12-Aug-14  2014    Unprovoked   
305     2014.07.22             22-Jul-14  2014    Unprovoked   
323   2014.06.09.a             09-Jun-14  2014    Unprovoked   
347     2014.04.03             03-Apr-14  2014    Unprovoked   
361   2014.02.07.b             07-Feb-14  2014      Provoked   
362     2014.02.07             07-Feb-14  2014    Unprovoked   
363     2014.01.26             26-Jan-14  2014      Provoked   
364     2014.01.25             25-Jan-14  2014    Unprovoked   
365     2014.01.04             04-Jan-14  2014  Sea Disaster   
367     2013.12.25             25-Dec-13  2013    Unprovoked   
377     2013.11.22             22-Nov-13  2013    Unprovoked   
378     2013.11.12             12-Nov-13  2013    Unprovoked   
379     2013.11.10             10-Nov-13  2013      Provoked   
380   2013.11.07.b             07-Nov-13  2013    Unprovoked   
381   2013.11.07.a             07-Nov-13  2013    Unprovoked   
382     2013.10.31             31-Oct-13  2013    Unprovoked   
383     2013.10.28             28-Oct-13  2013    Unprovoked   
384   2013.10.26.b             26-Oct-13  2013    Unprovoked   
448   2013.05.27.b             27-May-13  2013    Unprovoked   
449   2013.05.27.a             27-May-13  2013    Unprovoked   
823     2010.02.01             01-Feb-10  2010      Provoked   
1217  2006.08.22.b             22-Aug-06  2006       Invalid   
1218  2006.08.22.a             22-Aug-06  2006       Invalid   
2274    1993.04.00                Apr-93  1993       Invalid   
2477  1989.00.00.a                  1989  1989    Unprovoked   
3019    1975.01.19             19-Jan-75  1975    Unprovoked   
3549  1963.00.00.a                  1963  1963    Unprovoked   
3603  1962.03.25.b             25-Mar-62  1962      Provoked   
3917    1958.08.31             31-Aug-58  1958          Boat   
3928  1958.07.00.b                Jul-58  1958    Unprovoked   
4394    1945.07.30             30-Jul-45  1945  Sea Disaster   
4642    1936.08.04  Reported 04-Aug-1936  1936    Unprovoked   
4668    1935.08.24             24-Aug-35  1935       Invalid   
4719  1934.01.08.R  Reported 08-Feb-1934  1934       Boating   
5317    1900.09.05             05-Sep-00  1900    Unprovoked   
5458    1890.06.26            06-26-1890  1890    Unprovoked   
5686    1864.01.02           02-Jan-1864  1864    Unprovoked   
5694  1863.00.00.R         Reported 1863  1863    Unprovoked   
5819  1819.07.08.R  Reported 08-Jul-1819  1819       Invalid   
5857  1637.00.00.R         Reported 1637  1637    Unprovoked   

                Country                                        Area  \
20                SPAIN                           Alicante Province   
27              BAHAMAS                               Abaco Islands   
61                  USA                                     Florida   
107              BRAZIL                                  Pernambuco   
114                 USA                                     Florida   
134                 USA                                     

Hacemos lo mismo con Case Number.

In [33]:
df_aux[df_aux['CaseNumber_equals']==False]

Case Number                                  Date  Year        Type  \
4       2016.09.15                             16-Sep-16  2016  Unprovoked   
33    2016.07.14.4                 Reported  14-Jul-2016  2016  Unprovoked   
97    2016.01.24.b                             24-Jan-16  2016  Unprovoked   
116     2015.12.23                             07-Nov-15  2015     Invalid   
121   2015.10.28.a                             28-Oct-15  2015  Unprovoked   
3296    1967.07.05                             05-Jul-67  1967  Unprovoked   
3569  1962,08.30.b                             30-Aug-62  1962        Boat   
3654  1961.09.02.R                  Reported 06-Sep-1961  1961    Provoked   
4177    1952.08.05                             05-Aug-52  1952    Provoked   
5043  1900.00.00.R  Reported to have taken place in 1919  1919     Boating   
5150  1911.07.31.R                  Reported 31-Jul-1911  1911  Unprovoked   

        Country              Area                                   Location  \
4     AUSTRALIA          Victoria                                Bells Beach   
33      BAHAMAS               NaN                                Tiger Beach   
97          USA             Texas                               Off Surfside   
116         USA           Florida  Paradise Beach, Melbourne, Brevard County   
121         USA            Hawaii                               Malaka, Oahu   
3296     TURKEY    Mugla Province                            Kucukada Island   
3569     TURKEY  Antalya Province                                     Ucagiz   
3654      ITALY   Venice Province                                   Chioggia   
4177      ITALY            Teramo                                 Giulianova   
5043      ITALY               NaN                                     Savona   
5150      SPAIN           Málaga                                       Ceuta   

           Activity                                   Name Sex   \
4           Surfing                                   male    M   
33     Scuba Diving                      Michael Dornellas    M   
97     Spearfishing                             Keith Love    M   
116         Surfing                         Ryla Underwood    F   
121   Body boarding                        Raymond Senensi    M   
3296   Spearfishing                           Gungor Guven    M   
3569            NaN                   Occupant: Hasan Olta    M   
3654        Fishing  Pollione Perrini & Fioravante Perini     M   
4177        Fishing                         Vittorio Speca  NaN   
5043        Fishing                                    NaN    M   
5150        Bathing                              a soldier    M   

            ...                                Species   \
4           ...                               2 m shark   
33          ...                         Lemon shark, 9'   
97          ...                         Bull sharks x 2   
116         ...         Shark involvement not confirmed   
121         ...                                     NaN   
3296        ...                                     NaN   
3569        ...                                     NaN   
3654        ...                               1 m shark   
4177        ...                                2m shark   
5043        ...                               13' shark   
5150        ...                                     NaN   

           Investigator or Source                          pdf  \
4              The Age, 9/16/2016    2016.09.16-BellsBeach.pdf   
33             GrindTV, 7/14/2016  2016.07.14.R-TigerBeach.pdf   
97                        K. Love        2016.01.24.b-Love.pdf   
116       Fox25Orlando, 11/7/2015     2015.11.07-Underwood.pdf   
121   Star Advertiser, 10/28/2015       2015.10.28-Senensi.pdf   
3296               C. Moore, GSAF         1967.07.05-Guven.pdf   
3569                C.Moore, GSAF             1962.08.30.b-pdf   
3654               C. Moore, GSAF    1961.09.06.R-Chioggia.pdf   
4

In [40]:
df_aux[df_aux['Year'] == 0]

Case Number                                               Date  Year  \
5868      0.0214                                       Ca. 214 B.C.     0   
5869      0.0336                                      Ca. 336.B.C..     0   
5870      0.0493                                           493 B.C.     0   
5871      0.0725                                       Ca. 725 B.C.     0   
5872     ND-0153                                       1990 or 1991     0   
5873     ND-0152                                        Before 2016     0   
5874     ND-0151                                    Before Oct-2009     0   
5875     ND-0150                                        Before 1934     0   
5876     ND-0149                                        Before 1934     0   
5877     ND-0148                                              2009?     0   
5878     ND-1940                                        Before 1930     0   
5879     ND-0139                                          1880-1899     0   
5880     ND-0138                                        Before 1909     0   
5881     ND-0136                                        Before 2012     0   
5882     ND-0135                                        Before 1916     0   
5883     ND-0134                                Between   1951-1963     0   
5884     ND-0133                                        Before 1908     0   
5885     ND-0132                                        Before 1900     0   
5886     ND-0130                                        Before 1876     0   
5887     ND-0129                                        Before 2012     0   
5888     ND-0127                                        Before 2011     0   
5889     ND-0124                                        Before 2011     0   
5890     ND-0123                                        Before 2009     0   
5891     ND-0122                                       Beforer 1994     0   
5892     ND-0119                                        Before 1963     0   
5893     ND-0118                                          1896-1913     0   
5894     ND-0116                                        Before 1936     0   
5895     ND-0115                                 Before 08-Jun-1912     0   
5896     nd-0114                                        Before 2012     0   
5897     ND-0113                                        Before 1911     0   
...          ...                                                ...   ...   
5962     ND.0031                                       World War II     0   
5963     ND.0030                                        Before 1905     0   
5964     ND.0028                            A few years before 1938     0   
5965     ND.0027                                            No date     0   
5966     ND.0026                                        Early 1930s     0   
5967     ND.0025                                        Before 1927     0   
5968     ND.0024                                Between 1918 & 1939     0   
5969     ND.0023                                            No date     0   
5970     ND.0022                                            No date     0   
5971     ND.0021                                            No date     0   
5972     ND.0020                                         1920 -1923     0   
5973     ND.0019                                        Before 1921     0   
5974     ND.0018                                        Before 1911     0   
5975     ND.0017                                        Before 1921     0   
5976     ND.0016                                        Before 1921     0   
5977     ND.0015                                        Before 1917     0   
5978     ND.0014                        Before          17-Jul-1916     0   
5979     ND.0013  No date (3 days after preceding incident) & pr...     0   
5980     ND.0012                        Before          19-Jul-1913     0   
5981     ND.0011                                        Before 1911     0   
5

In [50]:
df_aux.Country.value_counts()

USA                             2116
AUSTRALIA                       1279
SOUTH AFRICA                     565
PAPUA NEW GUINEA                 133
NEW ZEALAND                      125
BRAZIL                           102
BAHAMAS                           98
MEXICO                            81
ITALY                             71
FIJI                              62
PHILIPPINES                       59
REUNION                           57
NEW CALEDONIA                     51
MOZAMBIQUE                        44
CUBA                              42
SPAIN                             40
INDIA                             37
EGYPT                             36
CROATIA                           34
PANAMA                            32
JAPAN                             32
IRAN                              29
SOLOMON ISLANDS                   29
GREECE                            25
HONG KONG                         24
JAMAICA                           23
FRENCH POLYNESIA                  22
I

In [56]:
#df_aux[df_aux['Country']==SPAIN]
df_aux[df_aux["Country"].isin(["SPAIN"])] 

Case Number                  Date  Year        Type Country  \
20      2016.07.29             29-Jul-16  2016  Unprovoked   SPAIN   
34    2016.07.08.R  Reported 08-Jul-2016  2016  Unprovoked   SPAIN   
104     2015.12.25             25-Dec-15  2015  Unprovoked   SPAIN   
159   2015.08.18.a             18-Aug-15  2015     Invalid   SPAIN   
249   2014.12.03.R  Reported 03-Dec-2014  2014    Provoked   SPAIN   
282     2014.09.00                Sep-14  2014  Unprovoked   SPAIN   
287   2014.08.27.c             27-Aug-14  2014  Unprovoked   SPAIN   
307     2014.07.20             20-Jul-14  2014  Unprovoked   SPAIN   
434     2013.07.09             07-Jul-13  2013     Invalid   SPAIN   
788   2010.07.16.b             16-Jul-10  2010    Provoked   SPAIN   
884   2009.07.24.a             24-Jul-09  2009     Invalid   SPAIN   
1234    2006.07.13             13-Jul-06  2006     Invalid   SPAIN   
2251    1993.09.03             03-Sep-93  1993  Unprovoked   SPAIN   
2596    1986.04.00                Apr-86  1986  Unprovoked   SPAIN   
2597    1986.03.18             18-Mar-86  1986  Unprovoked   SPAIN   
2834    1980.07.26             26-Jul-80  1980  Unprovoked   SPAIN   
3184  1970.00.00.g            Late 1970s  1970  Unprovoked   SPAIN   
3633  1962.00.00.c                  1962  1962  Unprovoked   SPAIN   
4808    1930.08.16             16-Aug-30  1930  Unprovoked   SPAIN   
4834    1929.10.02             02-Oct-29  1929  Unprovoked   SPAIN   
4918    1926.07.03             03-Jul-26  1926     Invalid   SPAIN   
4932    1925.09.04             04-Sep-25  1925    Provoked   SPAIN   
4956    1924.03.24             24-Mar-24  1924     Boating   SPAIN   
5076    1916.07.08             08-Jul-16  1916  Unprovoked   SPAIN   
5143    1912.00.00                  1912  1912  Unprovoked   SPAIN   
5150  1911.07.31.R  Reported 31-Jul-1911  1911  Unprovoked   SPAIN   
5185  1909.04.27.R  Reported 27-Apr-1909  1909    Provoked   SPAIN   
5196  1908.08.28.R  Reported 28-Aug-1908  1908  Unprovoked   SPAIN   
5197  1908.07.18.R  Reported 18-Jul-1908  1908     Invalid   SPAIN   
5581    1879.09.22           22-Sep-1879  1879    Provoked   SPAIN   
5683    1864.07.25           25-Jul-1864  1864  Unprovoked   SPAIN   
5698  1862.08.15.R  Reported 15-Aug-1862  1862  Unprovoked   SPAIN   
5699  1862.08.02.R  Reported 02-Aug-1862  1862     Invalid   SPAIN   
5700    1862.07.25           25-Jul-1862  1862  Unprovoked   SPAIN   
5701    1862.07.14           14-Jul-1862  1862  Unprovoked   SPAIN   
5702    1862.07.13           13-Jul-1862  1862  Unprovoked   SPAIN   
5794  1836.07.26.R          1836.07.26.R  1836     Invalid   SPAIN   
5819  1819.07.08.R  Reported 08-Jul-1819  1819     Invalid   SPAIN   
5842    1764.00.00                  1764  1764  Unprovoked   SPAIN   
5887       ND-0129           Before 2012     0  Unprovoked   SPAIN   

                     Area                               Location  \
20      Alicante Province                       Arenales del Sol   
34         Canary Islands                Las Teresitas, Tenerife   
104   Grand Canary Island   Arinaga Beach, Aguimes, Gran Canaria   
159              Alicante               Poniente Beach, Benidorm   
249               Granada                             Off Motril   
282             Catalonia                                  Salou   
287              Alicante                               Benidorm   
307        Canary Islands         Teresita, Santa Cruz, Tenerife   
434             Catalonia  Sant Marti d’Empuries Beach, L’Escala   
788   Grand Canary Island              Sardina del Norte, Gáldar   
884             Catalunya                          Sant Salvador   
1234             Alicante                       San Juan Beach     
2251         Costa Blanca         Playa de las Arenas, Valencia    
2596            Catalonia                          Lloret-de-Mar   
2597                Cádiz                                Tarifa    
2834       Canary Islands        San Juan de la 

In [71]:
df_aux.columns

Index(['Case Number', 'Date', 'Year', 'Type', 'Country', 'Area', 'Location',
       'Activity', 'Name', 'Sex ', 'Age', 'Injury', 'Fatal (Y/N)', 'Time',
       'Species ', 'Investigator or Source', 'pdf', 'href formula', 'href',
       'Case Number.1', 'Case Number.2', 'original order', 'pdf_equals',
       'CaseNumber_equals'],
      dtype='object')

In [76]:
df_aux.rename(columns = {'Sex ':'Sex'},inplace = True)

In [78]:
df_aux.Sex.value_counts()

M      4835
F       585
M         2
N         1
lli       1
.         1
Name: Sex, dtype: int64

In [87]:
df_aux.replace('M','Male',inplace = True)
df_aux.replace('F','Female',inplace = True)
df_aux.replace('N','-',inplace = True)
df_aux.replace('lli','-',inplace = True)
df_aux.replace('.','-',inplace = True)
df_aux.replace('M ','Male',inplace = True)

In [88]:
df_aux.Sex.value_counts()

Male      4837
Female     585
-            3
Name: Sex, dtype: int64

In [91]:
df_aux.fillna('-', inplace = True)

In [101]:
df_aux[df_aux['Year']== 77&5]

Case Number        Date  Year        Type    Country             Area  \
5867  0005.00.00  Ca. 5 A.D.     5  Unprovoked  AUSTRALIA  New South Wales   

     Location Activity  Name   Sex        ...        Species   \
5867    Bondi        -  male  Male        ...               -   

     Investigator or Source                                  pdf  \
5867        Waverly Library  0005.00.00-AustralianAboriginal.pdf   

                                           href formula  \
5867  http://sharkattackfile.net/spreadsheets/pdf_di...   

                                                   href Case Number.1  \
5867  http://sharkattackfile.net/spreadsheets/pdf_di...    0005.00.00   

     Case Number.2 original order pdf_equals CaseNumber_equals  
5867    0005.00.00            126       True              True  

[1 rows x 24 columns]